packages/sdk/src/page-utils.ts

```diff
@@ -73,3 +73,15 @@ export const getPagePath = (id: Folder["id"] | Page["id"], pages: Pages) => {
 
   return paths.reverse().join("/").replace(/\/+/g, "/");
 };
+
+/**
+ * convert URLSearchParams instance into key/value object
+ * to easily access from expressions
+ */
+export const convertSearchParams = (searchParams: URLSearchParams) => {
```

Comment:
Object.fromEntries doesn't work for search params? isn't this just a regular iterable? 🤯

Reply:
You are right